sagemaker-lineage/sagemaker-lineage-cross-account-with-ram.ipynb

```diff
@@ -104,7 +104,7 @@
     "\n",
```

Comment:
<p>Can you replace the user input with something like below:</p><pre class="ql-syntax" spellcheck="false">import boto3
sts = boto3.client("sts")
consumer_account_id = sts.get_caller_identity()["Account"]
</pre><p>It will remove the dependency from user input and enable the CI to run the notebook end to end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3047/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
This is a cross-account example, so the account ID being provided manually ("consumer" account) is different from the account that the code will be running in ("producer" account).